src/index.html

```diff
@@ -7,8 +7,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img class="logo" src="images/logo.png"
+        alt="Moyo" >
+      </a></img>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item"> <a class="nav_link is-active" href="#">Apple</a></li>
+          <li class="nav__item"> <a class="nav_link" href="#samsung">Samsung</a></li>
+          <li class="nav__item"> <a class="nav_link" href="#smartphones">Smartphones</a></li>
+          <li class="nav__item"> <a class="nav_link" href="#laptopsandcomputers"
+            data-qa="hover">Laptops & Computers</a>
+          </li>
+          <li class="nav__item"> <a class="nav_link" href="#gadgets">Gadgets</a></li>
+          <li class="nav__item"> <a class="nav_link" href="#tablets">Tablets</a></li>
+          <li class="nav__item"> <a class="nav_link" href="#photo">Photo</a></li>
+          <li class="nav__item"> <a class="last_link" href="#video">Video</a></li>
```

Comment:
```suggestion
          <li class="nav__item"> 
            <a class="nav__link" href="#video">Video</a>
          </li>
```
Don't hardcode the class name for last element. You can select it using pseudoclasses
https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes#tree-structural_pseudo-classes